### PR TITLE
add support for version 2018-05-22

### PR DESCRIPTION
**Pull request by r-ohan**
Adds support for api version 2018-05-22

### CODE REVIEW

File: plaid/requester.py [hunk 1]
```diff
@@ -34,8 +38,18 @@ def _requests_http_request(url, method, data, timeout=DEFAULT_TIMEOUT):
         )
 
 
-def http_request(url, method=None, data=None, timeout=DEFAULT_TIMEOUT):
-    response = _requests_http_request(url, method, data or {}, timeout)
+def http_request(
```
Comment (joyzheng) [inline plaid/requester.py]: let's keep the argument order consistent between `http_request` and `_requests_http_request` (the order of `headers` and `timeout` is reversed between the two).

Comment (r-ohan) [reply plaid/requester.py]: Great catch! You're right about the major release, my mistake.

**Comment by r-ohan** [comment]
Closing as we want to keep this functionality internal until the full versioning release.

**Comment by michaelckelly** [comment]
🐢

**Pull request merged**